rules/windows/execution_suspicious_psexesvc.toml

```diff
@@ -12,22 +12,19 @@ evade detection.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License"
 name = "Suspicious Process Execution via Renamed PsExec Executable"
 risk_score = 47
 rule_id = "e2f9fdf5-8076-45ad-9427-41e0e03dc9c2"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
 timeline_id = "76e52245-7519-4251-91ab-262fb1a1728c"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  (process.pe.original_file_name:(psexesvc.exe or PSEXESVC.exe) or
-     winlog.event_data.OriginalFileName:(psexesvc.exe or PSEXESVC.exe)) and
-  process.parent.name:services.exe and
-  not process.name:(psexesvc.exe or PSEXESVC.exe)
+process where event.type in ("start", "process_started", "info") and
+  process.pe.original_file_name == "psexesvc.exe" and process.name != "PSEXESVC.exe"
```

Comment:
do we want to add `process.parent.name:services.exe`? Otherwise, looks good.

Reply:
not necessary, psexesvc is dropped at runtime (from psexec PE resource section), avoid also instances where parent.process.name is not captured (null),